docs: add the description of config.yaml support environment variables (#8901)
apache/apisix: docs/en/latest/admin-api.md

            port: 9180                  # Specific port, which must be different from node_listen's port.
```

> ### Support environment variables

Reviewer: Maybe change this title to "Using environment variables"?

> To configure via environment variables, you can use `${{VAR}}` syntax. For instance:

Reviewer: ```suggestion
To configure via environment variables, you can use the `${{VAR}}` syntax. For instance:
```

```yaml title="./conf/config.yaml"
deployment:
  admin:
    admin_key:
    - name: admin
      key: ${{ADMIN_KEY}} 
      role: admin
    allow_admin:  
    - 127.0.0.0/24
    admin_listen:
      ip: 0.0.0.0            
      port: 9180                  
```

>  And then run `export ADMIN_KEY=$your_admin_key` before `make init`.

Reviewer: ```suggestion
 And then run `export ADMIN_KEY=$your_admin_key` before running `make init`.
```

 If the configured environment variable can't be found, an error will be thrown.

>  Also, If you want to use default value when the environment variable not set,

Reviewer: ```suggestion
 If you want to use a default value when the environment variable is not set,
```

>  Use `${{VAR:=default_value}}` instead. For instance:

Reviewer: ```suggestion
 use `${{VAR:=default_value}}` instead. For instance:
```

Add this to the previous sentence.

```yaml title="./conf/config.yaml"
deployment:
  admin:
    admin_key:
    - name: admin
      key: ${{ADMIN_KEY:=edd1c9f034335f136f87ad84b625c8f1}} 
      role: admin
    allow_admin:  
    - 127.0.0.0/24
    admin_listen:
      ip: 0.0.0.0            
      port: 9180                  
```

> This will find environment variable `ADMIN_KEY` first, and if it's not exist it will use `edd1c9f034335f136f87ad84b625c8f1` as default value.

Reviewer: ```suggestion
This will find the environment variable `ADMIN_KEY` first, and if it does not exist, it will use `edd1c9f034335f136f87ad84b625c8f1` as the default value.
```
